app/helpers/review_mapping_helper.rb

```diff
@@ -0,0 +1,121 @@
+module ReviewMappingHelper
+
+    def create_report_table_header(headers = {})
+        table_header = "<div class = 'reviewreport'>\
+                        <table width='100% cellspacing='0' cellpadding='2' border='0'>\
+                        <tr bgcolor='#CCCCCC'>"
+        headers.each do |header, percentage|
+            table_header += "<th width = #{percentage}>\
+                            #{header.humanize}\
+                            </th>"
+        end
+        table_header += "</tr>"
+        table_header.html_safe
+    end
+    #
+    # for review report
+    #
+    def get_data_for_review_report(reviewed_object_id, reviewer_id, type, line_num)
```

Comment:
Assignment Branch Condition size for get_data_for_review_report is too high. [25.18/15]